### DISCUSSION TITLE
alphas on thumb clusters?

### DISCUSSION THREAD

**Discussion by masters3d** [binarybottle/engram-layout#60] [Ideas]
I recently got a glove80 with more accessible thumb clusters. I am thinking of moving 2-4 keys to the thumbs. Any suggestion on which keys might be best on thumbs? 

I looked at some thumb based layouts and most of them move e,t or s n h. I was thinking maybe I can free up the pinkies by moving n c.

**Comment by sunaku** [comment]
To clarify, the keyboard in question is the *Glove80* whose physical key arrangement and logical key layout, respectively, are as follows:

![Glove80 dimensions](https://cdn.shopify.com/s/files/1/0695/7078/9649/files/Dimensions_1800x.jpg?v=1672298157)
![Glove80 key layout](https://my.glove80.com/images/keymap_glove80.png)
